package.json

```diff
@@ -6,7 +6,7 @@
   "author": "Kevin Barabash <kevinb7@gmail.com>",
   "license": "MIT",
   "dependencies": {
-    "math-evaluator": "^0.0.1",
+    "math-evaluator": "^0.0.2",
```

Comment:
When updating deps you should do `yarn upgrade math-evaluator` so that the yarn.lock file is updated.  That way we're both using exactly the same deps (and sub-deps).